ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -1104,33 +1099,6 @@ func (r *RayServiceReconciler) reconcileServices(ctx context.Context, rayService
 	return nil, err
 }
 
-func (r *RayServiceReconciler) updateStatusForActiveCluster(ctx context.Context, rayServiceInstance *rayv1.RayService, rayClusterInstance *rayv1.RayCluster) error {
-	logger := ctrl.LoggerFrom(ctx)
-	rayServiceInstance.Status.ActiveServiceStatus.RayClusterStatus = rayClusterInstance.Status
-
-	var err error
-	var clientURL string
-	rayServiceStatus := &rayServiceInstance.Status.ActiveServiceStatus
-
-	if clientURL, err = utils.FetchHeadServiceURL(ctx, r.Client, rayClusterInstance, utils.DashboardPortName); err != nil || clientURL == "" {
-		return err
-	}
-
-	rayDashboardClient := r.dashboardClientFunc()
-	if err := rayDashboardClient.InitClient(ctx, clientURL, rayClusterInstance); err != nil {
-		return err
-	}
-
-	var isReady bool
-	if isReady, err = getAndCheckServeStatus(ctx, rayDashboardClient, rayServiceStatus); err != nil {
```

Comment:
This will set the Ray Serve application status in the CR status. However, it is ineffective.

1. If the pending cluster is not ready, `Status().Update(...)` will not be called.
2. If the pending cluster is ready, it will be promoted to the active cluster. Therefore, the Ray Serve application status set here will be overwritten.